numba/tests/test_array_exprs.py

```diff
@@ -627,5 +644,127 @@ def do_call(x, y):
         self.assertTrue(isinstance(oty.type.dtype, types.Float))
 
 
+class TestExternalTypes(MemoryLeakMixin, unittest.TestCase):
+    """ Tests RewriteArrayExprs with external (user defined) types,
+    see #5157"""
+
+    source_lines = """
```

Comment:
would suggest to use `textwrap.dedent` for better code alignment here: https://docs.python.org/3/library/textwrap.html#textwrap.dedent

Reply:
done
thanks